packages/dev/docs/pages/react-aria/testing.mdx

```diff
@@ -0,0 +1,109 @@
+within the component or to verify that the component's state has changed appropriately post-interaction. By providing utilities to simulate these standard interactions and getters that
+allow the user to easily look up the subcomponents of the component itself, we hope to simplify the overall test writing experience, leading to easier adoption.
+
+These test utilities were inspired by various issues and observations that the maintainers of this library and consumers have experienced when writing tests against our components over the years. It is still very much
+a work in progress so if you discover any issues or have any feedback, please feel free to report them via [GitHub issues](https://github.com/adobe/react-spectrum/issues)! If you have implemented
+any testing utilities yourself that you feel would be a good fit, we would be happy to review any pull requests! Please read our [contributing guide](contribute.html)
+for more information.
+
+### Installation
+
+`@react-aria/test-utils` can be installed using a package manager like [npm](https://docs.npmjs.com/cli/npm) or [yarn](https://classic.yarnpkg.com/lang/en/).
+
+```
+yarn add --dev @react-aria/test-utils
+```
+
+Please note that this library uses [@testing-library/react@15](https://www.npmjs.com/package/@testing-library/react) and [@testing-library/user-event](https://www.npmjs.com/package/@testing-library/user-event/v/13.1.5). This means that you need
+to be on React 18+ in order for these utilities to work.
+
+### Setup
+
+Once installed, you can access the `User` that `@react-aria/test-utils` provides in your test file as shown below. This user only needs to be initialized once and accepts two options: `interactionType` and `advanceTimer`. `interactionType` will
+initialize what mode of interaction (mouse, keyboard, or touch) will be used by default. This can be overridden at the pattern tester or interaction execution level if required. `advanceTimer` accepts a function that when called should advance timers (fake timers)
+or wait for the required time (real timers). This is required for certain interactions (e.g. long press) that some of the patterns support.
+
+Once the `User` is initialized, you can use its `createTester` method to initialize a specific ARIA pattern tester in your test cases. This gives you access to that pattern's specific utilities that you can then call
+within your test to query for specific subcomponents or simulate common interactions. `createTester` requires two arguments, the first being the name of the ARIA pattern tester you are creating and the second being a set of initialization options specific to that
+pattern, typically including the `root` element (e.g. the menu trigger button, table, etc). See [below](#patterns) for more details on what is supported for each individual ARIA pattern tester.
+
```

Comment:
We may be able to reduce the amount of description here as well. Maybe just a smaller introduction, and then some inline comments in the code example below to describe the options (e.g. interactionType)?